blockchain/state/doc.go

```diff
@@ -18,5 +18,33 @@
 // This file is derived from core/state/statdb.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package state provides a caching layer atop the Klaytn state trie.
+/*
+Package state provides a uppermost caching layer of the Klaytn state trie.
+This package is used to read/write stateObject from/to StateDB and it also acts as a bridge between
+the objects and the persistent layer.
+
+Overview of state package
+
+stateObject represents a Klaytn account identified by its address.
+Once it is loaded from the persistent layer, it is cached and managed by StateDB.
```

Comment:
```suggestion
Once it is loaded from the persistent layer, it is cached and managed by StateDB.

```
A blank line for readability

Reply:
Added!